extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsExtractor.java

```diff
@@ -163,8 +222,28 @@ private String getDataString(Map<String, String> params) throws UnsupportedEncod
     }
 
     private String findValue(final String doc, final String start, final String end) {
```

Comment:
Note for a future PR: this should be moved to a separate string utils class and be tested

Reply:
→ https://github.com/TeamNewPipe/NewPipeExtractor/issues/679